index.html

```diff
@@ -6,128 +6,133 @@
 
-        <h3 class="navelement"><a href="#skills">Skills</a> -</h3>
+        <li class="nav__element"><a href="#skills">Skills</a> -</li>
 
-        <h3 class="navelement"><a href="#links">Links</a> -</h3>
+        <li class="nav__element"><a href="#links">Links</a> -</li>
 
-        <h3 class="navelement"><a href="#kontakt">Contact</a></h3>
+        <li class="nav__element"><a href="#kontakt">Contact</a></li>
 
-        <div class="arrow" id="arrow">
-          <img class="arrowimg" src="bilder/arrow.gif" alt="" />
+        <div class="nav__arrow" id="arrow">
+          <img class="nav__arrow-img" src="bilder/arrow.gif" alt="Arrow" />
         </div>
-      </div>
+      </ul>
     </nav>
     <main>
-      <div class="containerdiv background-welcome positionrelative">
+      <section class="containerdiv background-welcome positionrelative">
         <br /><br />
         <br /><br />
         <figure align="center" class="figure">
-          <img src="/bilder/lenz.jpg" class="image" />
+          <img
+            src="/bilder/lenz.jpg"
+            class="lenz-image"
+            alt="Lenz Lösche sitting at desk"
+          />
         </figure>
-      </div>
+      </section>
       <div class="line"></div>
 
-      <div class="containerdiv background-vita" align="center" id="lebenslauf">
-        <div class="vitatable">
-          <div class="about">
+      <section
+        class="containerdiv background-vita"
+        align="center"
+        id="lebenslauf"
+      >
+        <article class="about">
+          <div class="about__text">
             <p>
               Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ipsam
               iure esse asperiores quia eius dolorum similique repudiandae
               magnam tenetur cupiditate perspiciatis est quis illum expedita,
               quibusdam voluptates possimus quod ducimus!
-            </p>
-            <p>
+              <br />
               Lorem ipsum dolor sit amet consectetur adipisicing elit. Error ex
               a, perferendis necessitatibus nam porro minima reiciendis? Illo
               ipsam repellendus esse recusandae dignissimos dolores! Placeat
               blanditiis reprehenderit dignissimos voluptas. Voluptatibus!
             </p>
           </div>
-        </div>
-      </div>
+        </article>
+      </section>
       <div class="line"></div>
-      <div class="containerdiv background-skills" id="skills">
-        <div class="skillsgrid">
-          <div class="skillitem"><p>HTML</p></div>
-          <div class="skillitem"><p>CSS</p></div>
-          <div class="skillitem"><p>JavaScript</p></div>
-          <div class="skillitem"><p>React</p></div>
-          <div class="skillitem"><p>Git</p></div>
-          <div class="skillitem"><p>Github</p></div>
-          <div class="skillitem"><p>CleanCode</p></div>
+      <section class="containerdiv background-skills" id="skills">
+        <div class="skill">

```

Comment:
semantically this looks like a list - please consider to use `<ul>` instead of `<div>`